templates/17_upsert.tpl

```diff
@@ -39,7 +39,7 @@ func (o *{{$tableNameSingular}}) Upsert(exec boil.Executor, {{if eq .DriverName
 
 	// Build cache key in-line uglily - mysql vs psql problems
 	buf := strmangle.GetBuffer()
-	{{if eq .DriverName "psql"}}
+	{{if (or (eq .DriverName "psql") (eq .DriverName "crdb"))}}
```

Comment:
Ignoring the upsert portions of this stuff, these conditionals are really hard for out of band drivers. Since we do have a lot of conditional logic that says "if postgres" or "if mysql". We'll probably have to do some additional work to clean these up into more abstract pieces or the entire system will suffer a bit.